lib/omnisharp-atom/linter.ts

```diff
@@ -84,6 +84,13 @@ class LinterCSharp extends Linter.Linter {
             });
         }
     }
+
+    private getFrameworks(error: OmniSharp.Models.DiagnosticLocation): string {
+        var frameworks = _.map(error.Projects, (project: string) => {
+            return project.split('+')[1];
+        }).filter((fw: string) => fw.length > 0);
+        return frameworks.join(',');
+    }
```

Comment:
`OmniSharp.Models.DiagnosticLocation` is just an interface (because it's something that Omnisharp gives to us) so it doesn't really exist to add a method too (nor should we really be doing that).

We could add a help method onto `Omni` called `getFrameworks` that looks like:

```
function getFrameworks(container: { Projects: string[]; }) {...}
```

That way we can reuse it on any interface that has a projects collection.